braindecode/models/eegresnet.py

```diff
@@ -18,150 +18,226 @@ class EEGResNet(nn.Sequential):
     """
     Residual Network for EEG.
     """
-    def __init__(self, in_chans,
-                 n_classes,
-                 input_time_length,
-                 final_pool_length,
-                 n_first_filters,
-                 n_layers_per_block=2,
-                 first_filter_length=3,
-                 nonlinearity=elu,
-                 split_first_layer=True,
-                 batch_norm_alpha=0.1,
-                 batch_norm_epsilon=1e-4,
-                 conv_weight_init_fn=lambda w: init.kaiming_normal_(w, a=0)):
+
+    def __init__(
+        self,
+        in_chans,
+        n_classes,
+        input_time_length,
+        final_pool_length,
+        n_first_filters,
+        n_layers_per_block=2,
+        first_filter_length=3,
+        nonlinearity=elu,
+        split_first_layer=True,
+        batch_norm_alpha=0.1,
+        batch_norm_epsilon=1e-4,
+        conv_weight_init_fn=lambda w: init.kaiming_normal_(w, a=0),
+    ):
```

Comment:
is it some some form of automatic tool that did this reformating? If so I am not super happy as it makes large diffs to review. If we go this route then it should be made part of the CI system so formatting is not anymore a matter of taste but enforced why a system we agree on.

Reply:
@agramfort yes, the formatting was done automatically with [black](https://github.com/psf/black). I agree that this way of writing function calls is not space efficient and also doesn't look that good. I think it would make sense to either modify black's settings (if that's a thing we can actually do) or use some other tool, e.g. autopep8. @gemeinl @robintibor what do you think?